txnsync/bitmask.go

```diff
@@ -109,7 +111,7 @@ func (b *bitmask) trimBitmask(entries int) {
 	default:
 	}
 
-	*b = bytes.TrimRight(*b, "\x00")
+	*b = (*b)[:bestSize]
```

Comment:
nit: have this in case 0 and case 1, and delete `return` from case 2 and case 3. Also delete case `default`.